src/graphql/v2/dataSource.js

```diff
@@ -413,6 +407,22 @@ class FhirDataSource extends DataSource {
         }
         return combinedMeta;
     }
+
+    /**
+     * Concatenate code and/or display
+     * @param {Coding | undefined} foundCombinedMetaTag
+     * @param {Coding | undefined} metaTag
+     * @returns {Object}
+     */
+    updateCombinedMetaTag(foundCombinedMetaTag, metaTag) {
```

Comment:
Any function should be written in isolation from the various contexts it can be called in. foundCombinedMetaTag and metaTag are currently referring to the data used by the callers. Instead the parameter name should be as per the functionality like naming sourceMetaTag and targetMetaTag.